docs/EN/Configuration/xdrip.md

```diff
@@ -0,0 +1,238 @@
+
+## Basic settings for all CGM & FGM systems
+
+* Make sure to set Base URL correctly including <font color="#FF0000"><b>S</font></b> at the end of http<b><font color="#FF0000">s</font></b>:// (not http://)
+
+   i.e. https://API_SECRET@your-app-name.herokuapp.com/api/v1/
+
+   -> Hamburger Menu (top left of homescreen) -> Settings-> Cloud Upload-> Nightscout Sync (REST-API) -> Base URL
+* Deactivate “Automatic Calibration”
+   If the checkbox for "Automatic Calibration" is checked, activate "Download data" once, then remove the checkbox for "automatic calibration" and deactivate "Download data" again, otherwise the treatments (insulin & carbs) will be added twice to Nightscout.
+* Tap "Extra Options"
+* Deactivate "Upload treatments" and "Back-fill data"
+* Option "Alert on failures" should also be deactivated. Otherwise you will get an alarm every 5 minutes in case wifi/mobile network is too bad or the server is not available.
+
+   ![xDrip+ Basic Settings 1](../images/xDrip_Basic1.png)
+
+   ![xDrip+ Basic Settings 2](../images/xDrip_Basic2.png)
+
+* **InterApp-Settings** (Broadcast)
+   If you are going to use a DIY closed loop system and the data should be forwarded to i.e. AndroidAPS you have to activate broadcasting in xDrip+ in Inter-App settings.
+* In order for the values to be equal, you should activate "Send the displayed glucose value".
+* If you have also activated "Accept treatments" and broadcasting in AAPS, then xDrip+ will receive insulin, carbs and basal rate information from your DIY closed loop system and can estimate the hypo prediction etc. more accurately.
+
+   ![xDrip+ Basic Settings 3](../images/xDrip_Basic3.png)
+   
+## XDrip+ & Dexcom G6
+
+### Dexcom specific settings
+
+* Open G5/G6 Debug Settings 
+   -> Hamburger Menu (top left of homescreen)
+   -> Settings
+   -> G5/G6 Debug Settings
+   ![Open xDrip+ Settings](../images/xDrip_Dexcom_SettingsCall.png)
+   
+* Enable the following settings
+   - Use the OB1 Collector
+   - Native Algorithm (important if you want to use SMB)
+   - G6 support
+   - Allow OB1 unbonding
+   - Allow OB1 initiate bonding
+* All other options should be disabled
+* Adjust battery warning level to 280 (bottom of G5/G6 Debug Settings)
+
+   ![xDrip+ G5/G6 Debug Settings](../images/xDrip_Dexcom_DebugSettings.png)
+
+### Preemptive restarts not recommended
+
+The automatic extension of Dexcom sensors (“preemtive restarts”) is not recommended as this might lead to “jumps” in BG values on day 9 after restart.
+
+   ![xDrip+ Jump after Preemptive Restart](../images/xDrip_Dexcom_PreemptiveJump.png)
+
+What’s clear is that using the G6 is perhaps a little more complex than it as first suggests. To use it safely, there are a few points to be aware of: 
```

Comment:
duplicate with BG source text?

Reply:
On purpose. Some details are doubled as you cannot be sure where users search for information.